falcon/stream.py

```diff
@@ -14,11 +14,18 @@
 
 """WSGI BoundedStream class."""
 
+from __future__ import annotations
 import io
```

Comment:
This seems to cause our `pep8` CI gate to fail, something about import groups and their order.

Reply:
Done.